server/src/main/java/org/opensearch/action/support/TransportAction.java

```diff
@@ -79,6 +81,10 @@ private Releasable registerChildNode(TaskId parentTask) {
         }
     }
 
+    protected Tracer getTracer() {
```

Comment:
Please remove that, the `TransportAction` does not need tracer (at the moment), only concrete transport action need them